packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -16,8 +16,9 @@ export * from './startPSM.js';
  * @property {string} [issuerBoardId]
  * @property {string} [denom]
  * @property {number} [decimalPlaces]
- * @property {string} [proposedName]
+ * @property {string} proposedName
  * @property {string} keyword
+ * @property {string} [issuerName]
  * @property {string} oracleBrand
```

Comment:
why allow oracleBrand to be different than issuerName?

Reply:
I don't recall. Perhaps no reason. I suppose the thought at the time of writing was: why constrain them to be the same?

@michaelfig ? maybe you have a more clear idea of the constraints?

Reply:
oh... are you suggesting that `oracleBrand` suffices instead of adding an `issuerName` property? I suppose that could work, though it seems like a stretch.

Reply:
if they don't truly need to be different my suggestion would be to rename the `oracleBrand` property to `issuerName`.

Whether they need to be different… they are different brands but I think it's a source of unnecessary confusion to have different strings, since their different namespaces suffice.